src/main/java/zoe/Task.java

```diff
@@ -0,0 +1,64 @@
+package zoe;
+
+/**
+ * Superclass for all forms of tasks
+ * Contains some global methods so that all subclasses have access to that method
+ */
+public class Task {
+    protected String description;
+    protected String type;
+    protected boolean isDone;
+
+    public Task() {
+        this.description = "";
+        this.type = "";
+        this.isDone = false;
+    }
+
+    public Task(String description) {
+        this.description = description;
+        this.type = "";
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "X" : " ");
```

Comment:
Maybe you can consider using this.isDone as opposed to isDone to be more consistent with styling 
```suggestion
        return (this.isDone ? "X" : " ");
```